src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,170 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
+    static class Node<T> {
+        private T value;
+        private Node<T> next;
+        private Node<T> previous;
+
+        public Node(T value) {
+            this.value = value;
+        }
+
+        public Node(T value, Node<T> next, Node<T> previous) {
+            this.value = value;
+            this.next = next;
+            this.previous = previous;
+        }
+    }
```

Comment:
```suggestion
    static class Node<T> {
        private T value;
        private Node<T> next;
        private Node<T> previous;

        public Node(T value) {
            this.value = value;
        }

        public Node(T value, Node<T> next, Node<T> previous) {
            this.value = value;
            this.next = next;
            this.previous = previous;
        }
    }
   
```
 put class Node to the bottom of the class MyLinkedList